media/multipart/src/main/java/io/helidon/media/multipart/MultiPartDecoder.java

```diff
@@ -102,11 +103,10 @@ public void subscribe(Subscriber<? super ReadableBodyPart> subscriber) {
 
             @Override
             public void request(long n) {
-                long curr = n <= 0 ?
-                        partsRequested.getAndSet(-1) :
-                        partsRequested.getAndUpdate(v -> Long.MAX_VALUE - v > n ? v + n
-                                : v < 0 ? v
-                                : Long.MAX_VALUE);
+                long curr = n <= 0
+                        ? partsRequested.getAndSet(-1)
+                        : partsRequested.getAndUpdate(v -> Long.MAX_VALUE - v > n
+                        ? v + n : v < 0 ? v : Long.MAX_VALUE);
```

Comment:
This is a ternary operator inside a lambda, so you may want to indent it a bit more